source/args.c

```diff
@@ -233,7 +233,7 @@ int parse_args(struct args *args, int argc, const char *argv[])
 		{ "runs", required_argument, 0, OPTION_RUNS },
 		{ "threshold", required_argument, 0, OPTION_THRESHOLD },
 		{ "output", required_argument, 0, OPTION_OUTPUT },
-		{ 0 },
+		{ NULL, 0, 0, 0 },
```

Comment:
Ah, all right :). Could you also change this for source/arm/paging.c and source/arm64/paging.c? The changes should be pretty much identical for those files as well.